app/views/sign_up/select_email/show.html.erb

```diff
@@ -0,0 +1,46 @@
+<% self.title = t('titles.select_email') %>
+
+<%= render StatusPageComponent.new(status: :info, icon: :question) do |c| %>
+  <% c.with_header { t('titles.select_email') } %>
+  <p class="margin-top-4">
+    <%= I18n.t('help_text.select_preferred_email', sp: @sp_name, app_name: APP_NAME) %>
+  </p>
+
+  <%= simple_form_for('', url: sign_up_select_email_path) do |f| %>
+    <% @user_emails.each do |email, index| %>
+      <fieldset class="usa-fieldset">
+        <div class="usa-radio">
+          <%= radio_button_tag(
+                'select_email_form[selection]',
+                email,
+                email == @last_sign_in_email_address,
+                class: 'usa-radio__input usa-radio__input--bordered',
+              ) %>
+              <%= label_tag(
+                    "select_email_form_selection_#{email}",
+                    class: 'usa-radio__label width-full',
+                  ) do %>
+                <%= email %>
+              <% end %>
+        </div>
+      </fieldset>
+    <% end %>
+    <%= f.submit t('help_text.requested_attributes.change_email_link'), class: 'margin-top-4' %>
+  <% end %>
+
+   <div class="grid-row">
+    <div class="grid-col-6">
+    <%= render ButtonComponent.new(
+          url: add_email_path,
+          outline: true,
+          icon: :add,
+          class: 'margin-y-2 usa-button--big usa-button--wide',
+        ).with_content(t('account.index.email_add')) %>        
+    </div>
+  </div>
```

Comment:
-  What's the grid used for here? Typically I'd just expect big & wide button for these sorts of pages.
- `big` and `wide` can be passed as properties of the button constructor, so we don't have to deal with implementation detail of the specific CSS class modifiers. Same as what we do with `outline`
- `StatusPageComponent` also has a `with_action_button` helper for rendering buttons
- Is the icon expected? I don't see it in the Figma design reference.

```suggestion
  <% c.with_action_button(
     url: add_email_path,
     outline: true,
     big: true,
     wide: true,
   ).with_content(t('account.index.email_add')) %>
```

Reply:
I was getting a weird result with c.with_action_button - the markup was out of order and injected the button after the back link would show. Not sure how I thought the grid would help tbh but I saw it used in a similar situation.
It seems to be working now using a rendered ButtonComponent.